src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -194,11 +210,71 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             String moduleCode = split[1];
             return new DeleteModuleCommand(moduleCode);
 
+
+
         default:
             throw new IllegalCommandException();
         }
     }
 
+    /**
+     * Checks if word given is a valid command in Apollo
+     * @param word
+     * @return
+     */
+    public static boolean isValidWord(String word){
```

Comment:
Actually this method is not needed! You've already handled this in `chooseHelpCommand()`